core/src/main/java/org/kiwix/kiwixmobile/core/extensions/ViewExtensions.kt

```diff
@@ -48,29 +48,3 @@ fun View.snack(
     }
   }.show()
 }
-
-@SuppressLint("ShowToast")
-fun View.snack(
```

Comment:
Please check its usage before removing anything. Don't remove this function we are using this to show the message see https://github.com/kiwix/kiwix-android/blob/3f19f55b99c83152d9f6be2d910894c9409c8637/app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt#L539

Reply:
Sorry, for the inconvenience. I am fixing it rn.